.github/workflows/nix-aiopusher.yml

```diff
@@ -0,0 +1,18 @@
+name: aiopusher
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
```

Comment:
Instead of push we can make this part of our nocodb release flow so it will follow our release cycle.